test/addons_test.go

```diff
@@ -74,12 +77,57 @@ func init() {
 	}
 
 	fmt.Println("finding addon test groups...")
-	groups, err = testgroups.AddonsForGroupsFile("groups.yaml", cat)
+	groupsMap, err := getGroupsMapFromFile("groups.yaml")
+	if err != nil {
+		panic(err)
```

Comment:
Looks like we were doing panics here before, but is now potentially a good time for us to remove panics in this area in favor of something idiomatic for the tests?

Reply:
hmm... it's in the `init()` function, so i think it's appropriate for `panic` for now. think we'll save large refactors until we have the test framework ready.